content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -8,8 +8,8 @@ weight: 10
 
 This page will show you how to get started with OpenTelemetry in Node.js.
 
-You will learn how you can instrument a simple application automatically, in
-such a way that [traces][], [metrics][] and [logs][] are emitted to the console.
+You will learn how you can instrument a simple application, in such a way that
+[traces][] and [metrics][] are emitted to the console.
```

Comment:
Please incorporate that into #3302 to avoid conflict

Reply:
I predict that this PR can be merged sooner as https://github.com/open-telemetry/opentelemetry.io/pull/3302 is has no updates since 5 days.

Reply:
Done https://github.com/open-telemetry/opentelemetry.io/pull/3308/commits/0ead7683249b975a9ba5cfb67b1ff53178c58ea7